### DISCUSSION TITLE
library/uart

### DISCUSSION THREAD

**Discussion by giscus** [Air-duino/document#25] [comment]
# library/uart

关于 UART，即Universal Asynchronous Receiver/Transmitter（通用异步收发器），是一种串行、异步、全双工的通信协议，在嵌入式领域应用的非常广泛。 UART 通用API 在 Arduino 中，我们一般把UART称为Serial。我们在此仅介绍一些常用的 API ，具体的通用 API 可以参考 Arduino ...

https://arduino.luatos.com/library/uart.html

**Comment by old-Jerry** [comment]
air001-第二个串口波特率4800乱码，2400才能正常使用！！！注意

**Comment by lishuzhou** [comment]
请问第二个串口是否支持串口中断
